### ISSUE TITLE
Image sharing

### ISSUE THREAD

**Issue by SebastianZimmeck** [privacy-tech-lab/privacy-pioneer#482]
I am opening this issue to share images because MS Teams does not allows us to do so.

(cc'ing @harkous)

**Comment by SebastianZimmeck** [comment]
What does the following mean?

<img width="352" alt="Screenshot 2023-03-11 at 8 45 33 PM" src="https://user-images.githubusercontent.com/11034003/224519951-d91eb4bb-d5e8-4e32-b713-d83ac3f9051f.png">

This is the URL of our extension. Is this a mistake? I have not added any keyword. I am using the Firefox Add-on store version.

**Comment by SebastianZimmeck** [comment]
As to the question of whether it is possible to omit the extension URL, [there are extensions that do so](https://addons.mozilla.org/en-US/firefox/addon/gnome-download-notifications/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search) (lizard ... is the file name of the image I downloaded).

<img width="357" alt="Screenshot 2023-03-11 at 9 18 16 PM" src="https://user-images.githubusercontent.com/11034003/224520622-b86dd70d-7da3-49da-a78d-aba6d2b41994.png">

**Comment by SebastianZimmeck** [comment]
The watchlist count is 0 despite this being related to the watchlist. This is not such a big deal because there is no mention of the watchlist in the popup. So, people would possibly not associate the message with the watchlist.

<img width="1257" alt="Screenshot 2023-03-11 at 9 03 50 PM" src="https://user-images.githubusercontent.com/11034003/224520747-e8d5998a-12dd-46ee-8578-a09fa6e407f1.png">

**Comment by SebastianZimmeck** [comment]
The various usability problems here:

1. Most people will not recognize that this number refer to our extension.
2. It is confusing whether it is our extension that is collecting data or the site.
3. It is not clear which types of data (IP address and location) this refers to.
4. "Click the popup" should be "Click the extension icon."
5. Even when clicking the extension icon, people have no idea where they should be looking at.
6. The notification seems to come up on every site visited. This is both annoying and meaningless.

**Comment by lr-brown** [comment]
Potential Sample Notification: 

```
We found the following watchlist keywords in your web request: 
- Location (zipcode)
- General ("log...")
- Tracking (IP Address)
```

**Comment by SebastianZimmeck** [comment]
@lr-brown, yes, [that is excellent](https://github.com/privacy-tech-lab/privacy-pioneer/issues/482#issuecomment-1466217535). 

In particular, regarding

> Prompt Notification Permission on the toggle on of auto-generated keywords or after the first new keyword has been created

to keep it simple we ask for the notification permission once. From that moment on the notification permission stays active at the browser-level even if people disable notifications for some or all keywords. Such disabling is handled at the extension-level and leaves the browser-level notification permission as is.

**Comment by SebastianZimmeck** [comment]
> Potential Sample Notification:
> 
> ```
> We found the following watchlist keywords in your web request: 
> - Location (zipcode)
> - General ("log...")
> - Tracking (IP Address)
> ```

I would even keep it simpler and also use the exact same language as it is used on the Watchlist page:

```
We found the following Watchlist Keywords in the current web request: 
- IP Address
- Street Address
- General "log..."
```

**Comment by SebastianZimmeck** [comment]
@lr-brown, for

> Prompt Notification Permission on the toggle on of auto-generated keywords or after the first new keyword has been created

Here is a suggestion for the notification prompt language.

```
Privacy Pioneer will notify you of any selected Watchlist Keywords appearing in your web requests.
```

Then maybe some icon (or language) for disabling notifications (the same as in the individual notifications).

**Comment by SebastianZimmeck** [comment]
@danielgoldelman or @jjeancharles, can you review (and merge if everything checks out)?

**Comment by danielgoldelman** [comment]
@SebastianZimmeck I have to check a few things with my setup... getting some recurring problems that are due to either my Git setup, Firefox, or PP. Will do asap.
